register.html

```diff
@@ -25,11 +25,37 @@ <h1 class="logo">Relaxr</h1>
 				</nav>
 			</div>
 		</header>
-		<section id="main">
+		<section id="registerMain">
 			<div class="container">
-				<section id="contact">
-					<input type="submit" value="Register Now!">
-				</section>
+				<h2>Register for Relaxr!</h2>
+				<form id="registrationForm" novalidate>
+					<div id="errorMessage" class="error hide"></div>
+					<label for="username">Username: </label>
+					<input type="username" name="username"placeholder="Username">
```

Comment:
The `input`s need to have an `id` attribute that matches the `for` attribute. In other words:

```
<label for="username">Username: </label>
<input type="username" id="username" placeholder="Username">
```